frontend/src/features/admin-form/responses/ResponsesPage/storage/UnlockedResponses/ResponsesTable/ResponsesTable.tsx

```diff
@@ -193,7 +194,8 @@ const MRF_RESPONSE_TABLE_COLUMNS: Column<ResponseColumnData>[] = [
     Header: MRF_FIRST_STEP_TIMESTAMP_LABEL,
     accessor: 'submissionTime',
     width: 250,
-    disableResizing: true
+    minWidth: 250,
```

Comment:
Note: copying the same fix as existing to include minWidth which allows the bar to expand when the screen is small and horizontal scroll is enabled.